projects/schematics/src/migrations/5_0/constructor-deprecations/data/cart-lib.migration.ts

```diff
@@ -0,0 +1,2253 @@
+        importPath: '@spartacus/cart/base/root',
+      },
+    ],
+  },
+  {
+    class: 'NewSavedCartOrderEntriesContext',
+    importPath: '@spartacus/cart/saved-cart/root',
+    deprecatedParams: [
+      {
+        className: 'ActionsSubject',
+        importPath: '@ngrx/store',
+      },
+      {
+        className: 'UserIdService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'MultiCartService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'SavedCartFacade',
+        importPath: '@spartacus/cart/saved-cart/root',
+      },
+    ],
+    removeParams: [
+      {
+        className: 'ActionsSubject',
+        importPath: '@ngrx/store',
+      },
+      {
+        className: 'MultiCartService',
+        importPath: '@spartacus/core',
+      },
+    ],
+    addParams: [
+      {
+        className: 'ProductImportInfoService',
+        importPath: '@spartacus/cart/base/core',
+      },
+      {
+        className: 'MultiCartFacade',
+        importPath: '@spartacus/cart/base/root',
+      },
+    ],
+  },
+  {
+    class: 'SavedCartOrderEntriesContext',
+    importPath: '@spartacus/cart/saved-cart/root',
+    deprecatedParams: [
+      {
+        className: 'ActionsSubject',
+        importPath: '@ngrx/store',
+      },
+      {
+        className: 'UserIdService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'MultiCartService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'SavedCartFacade',
+        importPath: '@spartacus/cart/saved-cart/root',
+      },
+      {
+        className: 'RoutingService',
+        importPath: '@spartacus/core',
+      },
+    ],
+    removeParams: [
+      {
+        className: 'ActionsSubject',
+        importPath: '@ngrx/store',
+      },
+      {
+        className: 'MultiCartService',
+        importPath: '@spartacus/core',
+      },
+    ],
+    addParams: [
+      {
+        className: 'ProductImportInfoService',
+        importPath: '@spartacus/cart/base/core',
+      },
+      {
+        className: 'MultiCartFacade',
+        importPath: '@spartacus/cart/base/root',
+      },
+    ],
+  },
+  {
+    class: 'CartNotEmptyGuard',
+    importPath: '@spartacus/checkout/components',
+    deprecatedParams: [
+      {
+        className: 'ActiveCartService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'SemanticPathService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'Router',
+        importPath: '@angular/router',
+      },
+    ],
+    removeParams: [
+      {
+        className: 'ActiveCartService',
+        importPath: '@spartacus/core',
+      },
+    ],
+    addParams: [
+      {
+        className: 'ActiveCartFacade',
+        importPath: '@spartacus/cart/base/root',
+      },
+    ],
+  },
+  {
+    class: 'CheckoutAuthGuard',
+    importPath: '@spartacus/checkout/components',
```

Comment:
this one is also in checkout base components, and `UserAccountFacade` and `GlobalMessageService` are also removed. I think, that removal is caused by checkout refactor. Not sure whether other devs will update this migration doc.